src/compiler/diagnosticMessages.json

```diff
@@ -2477,5 +2477,10 @@
     "A type assertion expression is not allowed in the left-hand side of an exponentiation expression. Consider enclosing the expression in parentheses.": {
       "category": "Error",
       "code": 17007
+    },
+    "Potentially wrong empty statement in THEN clause.": {

```

Comment:
I would just say "Empty statement after conditional." or even "Empty statement after 'if'."  I don't like referencing internal concepts like `thenStatement` in errors.

@RyanCavanaugh, you seem to be the local expert on error message style. Can you take a look here too?